tools/hidl/AccessMapGenerator.hpp

```diff
@@ -0,0 +1,1003 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#ifndef _AccessMapGenerator_HPP_
+#define _AccessMapGenerator_HPP_
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdarg.h>
+#include <algorithm>
+#include <functional>
+#include <initializer_list>
+#include <list>
+#include <map>
+#include <string>
+#include <vector>
+
+#ifndef _WIN32
```

Comment:
You don't want to or can't include platform.h and use the (same I think) defines from there?